M2/Macaulay2/packages/Macaulay2Doc/functions/package-doc.m2

```diff
@@ -0,0 +1,718 @@
+     readPackage
+    (readPackage, String)
+    (readPackage, Package)
+    [readPackage, FileName]
+  Headline
+    read the package preamble
+  Usage
+    readPackage PackageName
+  Inputs
+    PackageName:String
+      the name of the package, or a @TO Package@ object
+    FileName=>String
+      the name of the file containing the source code of the package, from which it should be read
+  Outputs
+    :OptionTable
+      containing the options given to @TO newPackage@ in the preamble of the package
+  Description
+    Text
+      The file @TT "PackageName.m2"@ should be on the load @TO "path"@ and should contain a package named @TT "PackageName"@.
+      This function is mostly used for getting information about a package quickly and without processing the entire package.
+    Example
+      opts = readPackage "NormalToricVarieties"
+      "1.9" <= opts.Version
+  SeeAlso
+    Package
+    newPackage
+    loadPackage
+
+Node
+  Key
+     loadPackage
+    (loadPackage, String)
+    (loadPackage, Package)
+    [loadPackage, Configuration]
+    [loadPackage, DebuggingMode]
+    [loadPackage, FileName]
+    [loadPackage, LoadDocumentation]
+    [loadPackage, Reload]
+  Headline
+    load a package
+  Usage
+    loadPackage PackageName
+  Inputs
+    PackageName:String
+      the name of the package, or a @TO Package@ object
+    FileName=>String
+      the name of the file containing the source code of the package, from which it should be loaded
+    LoadDocumentation=>Boolean
+      whether to load the documentation of the package, too; see @TO beginDocumentation@
+    DebuggingMode=>Boolean
+      the value of @TO "debuggingMode"@ during loading the package; specifying it here overrides the
+      value specified as an option to @TO newPackage@ by the package itself; however, if @TO "debuggingMode"@
+      is already @TT "false"@ it will remain so.
+    Configuration=>List
+      the entries are options @TT "KEY => VALUE"@ overriding the defaults specified in the source code of the package
+      and the (possibly updated) values in the file in the user's application directory.
+    Reload=>Boolean
+      whether to reload the package, if it has been loaded before.
+      If the input was an already loaded @TO Package@, this is automatically set to true.
+  Outputs
+    :Package
+      the package just loaded; if the input was an already loaded @TO Package@, this is the reloaded package
+  Consequences
+    Item
+      Loads (or reloads) the package @TT "PackageName"@ that is in the file @TT "PackageName.m2"@.
+  Description
+    Text
+      The file @TT "PackageName.m2"@ should be on the load @TO "path"@ and should contain a package named @TT "PackageName"@.
+
+      If the variable @TO "notify"@ is set to true, then an informational message is displayed after the file is loaded.
+    Example
+      notify = true
+      loadPackage "FirstPackage"
+    Text
+      If that has been done, then this command will reload the package instead:
```

Comment:
This way of phrasing it makes it appear that setting "notify" to true is what makes the following command reload the package.
<img width="965" alt="Screen Shot 2020-09-08 at 9 09 55 AM" src="https://user-images.githubusercontent.com/700228/92480944-363eac00-f1ab-11ea-9eaf-846c706391a6.png">